payment_paysera/AUTHORS

```diff
@@ -0,0 +1,3 @@
+Authors ordered by first contribution.
+
+Naglis Jonaitis <hello@naglis.me>
```

Comment:
Please add yourself and/or Versada to the list. Also in `__manifest__.py` under `author`.